app/assets/stylesheets/styles/_events.scss

```diff
@@ -150,10 +187,17 @@
     ul {
       list-style-type: disc;
       list-style-position: inside;
-      columns: 3;
-      -webkit-columns: 3;
-      -moz-columns: 3;
-      margin-top: $margin;
+      margin-top: 2em;
+      columns: 2;
```

Comment:
This is really cool, however, there are some support issues. Might need to consider an alternate approach - http://caniuse.com/#feat=multicolumn

Also, just fyi, autoprefixer should be taking care of vendor prefixes for us, so you shouldn't have to add them yourself. (awesome sauce) We should probs check our output and make sure we're covering the prefixing we need. I'll look into that.